cheroot/errors.py

```diff
@@ -58,7 +67,8 @@ def plat_specific_errors(*errnames):
 
 acceptable_sock_shutdown_error_codes = {
     errno.ENOTCONN,
-    errno.EPIPE, errno.ESHUTDOWN,  # corresponds to BrokenPipeError in Python 3
+    errno.EPIPE,
```

Comment:
This one also corresponds to BrokenPipeError which is lost with the contextless formatting